code/modules/mob/living/simple_animal/abnormality/teth/lady_facing_the_wall.dm

```diff
@@ -0,0 +1,64 @@
+/mob/living/simple_animal/hostile/abnormality/wall_gazer
+	name = "Lady Facing the Wall"
+	desc = "An abnormality that is a pale, naked woman with long, black hair that completely obscures her face"
+	icon = 'ModularTegustation/Teguicons/96x48.dmi'
+	icon_state = "ladyfacingthewall"
+	maxHealth = 400
+	health = 400
+	threat_level = TETH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(55, 55, 55, 55, 55),
+		ABNORMALITY_WORK_INSIGHT = list(45, 45, 30, 30, 0),
+		ABNORMALITY_WORK_ATTACHMENT = list(100, 100, 100, 100, 100),
+		ABNORMALITY_WORK_REPRESSION = list(55, 55, 30, 30, 30)
+		)
+	pixel_x = -8
```

Comment:
Set to -32

Reply:
All done, The Lady now touches the wall.